mmv1/templates/terraform/examples/certificate_manager_certificate_map_entry_full.tf.erb

```diff
@@ -0,0 +1,39 @@
+resource "google_certificate_manager_certificate_map_entry" "<%= ctx[:primary_resource_id] %>" {
+  name        = "<%= ctx[:vars]['cert_map_entry_name'] %>"
+  description = "My acceptance test certificate map entry"
+   labels      = {
+    "terraform" : true,
+    "acc-test"  : true,
+ }
+  certificates = [google_certificate_manager_certificate.certificate.id]
+  matcher = "PRIMARY"
+}
+
+resource "google_certificate_manager_certificate" "certificate" {
+  name        = "<%= ctx[:vars]['cert_name'] %>"
+  description = "The default cert"
+  scope       = "EDGE_CACHE"
+  managed {
+    domains = [
+      google_certificate_manager_dns_authorization.instance.domain,
+      google_certificate_manager_dns_authorization.instance2.domain,
+      ]
+    dns_authorizations = [
+      google_certificate_manager_dns_authorization.instance.id,
+      google_certificate_manager_dns_authorization.instance2.id,
+      ]
+  }
+}
+
+
+resource "google_certificate_manager_dns_authorization" "instance" {
+  name        = "<%= ctx[:vars]['dns_auth_name'] %>"
```

Comment:
You'll also need `dns_auth_name` and `dns_auth_subdomain` (and the ones from `instance2`) all in `terraform.yaml` as well. You can look [here](https://github.com/GoogleCloudPlatform/magic-modules/blob/5081881c11f24659f88c8c1487c832be8fbbca8e/mmv1/products/certificatemanager/terraform.yaml#L39) for reference.